### ISSUE TITLE
Dependency warnings on install

### ISSUE THREAD

**Issue by emilyrose** [spark/spark-cli#167]
npm install -g spark-cli

> serialport@1.6.3 install /usr/local/lib/node_modules/spark-cli/node_modules/serialport
> node-pre-gyp install --fallback-to-build

[serialport] Success: "/usr/local/lib/node_modules/spark-cli/node_modules/serialport/build/serialport/v1.6.3/Release/node-v11-darwin-x64/serialport.node" is installed via remote
/usr/local/bin/spark -> /usr/local/lib/node_modules/spark-cli/bin/spark.js
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/npm-package-arg requires semver@'4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/semver,
npm WARN unmet dependency which is version 2.3.0
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/npm-registry-client/node_modules/npm-package-arg requires semver@'4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/semver,
npm WARN unmet dependency which is version 2.3.0
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/read-installed/node_modules/readdir-scoped-modules requires graceful-fs@'^3.0.4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/graceful-fs,
npm WARN unmet dependency which is version 3.0.2

Functionality seems fine but it's entirely possible that this could have unknown consequences.

**Comment by kennethlimcp** [comment]
Did a `sudo npm uninstall -g spark-cli` followed by `sudo npm install -g spark-cli --verbose`

```sh
KENMBP:libdw kennethlimcp$ sudo npm install -g spark-cli
 
> serialport@1.6.3 install /usr/local/lib/node_modules/spark-cli/node_modules/serialport
> node-pre-gyp install --fallback-to-build

[serialport] Success: "/usr/local/lib/node_modules/spark-cli/node_modules/serialport/build/serialport/v1.6.3/Release/node-v11-darwin-x64/serialport.node" is installed via remote
/usr/local/bin/spark -> /usr/local/lib/node_modules/spark-cli/bin/spark.js
spark-cli@1.0.0 /usr/local/lib/node_modules/spark-cli
├── xtend@4.0.0
├── node-wifiscanner@0.2.0
├── when@3.7.2
├── hogan.js@2.0.0
├── moment@2.9.0
├── glob@4.5.3 (inherits@2.0.1, inflight@1.0.4, once@1.3.1, minimatch@2.0.4)
├── request@2.54.0 (caseless@0.9.0, json-stringify-safe@5.0.0, aws-sign2@0.5.0, forever-agent@0.6.0, stringstream@0.0.4, oauth-sign@0.6.0, tunnel-agent@0.4.0, isstream@0.1.2, node-uuid@1.4.3, qs@2.4.1, form-data@0.2.0, mime-types@2.0.10, combined-stream@0.0.7, tough-cookie@0.12.1, http-signature@0.10.1, bl@0.9.4, hawk@2.3.1, har-validator@1.4.0)
└── serialport@1.6.3 (bindings@1.2.1, sf@0.1.7, async@0.9.0, debug@2.1.3, nan@1.7.0, optimist@0.6.1)
```

Seems ok on my side and i'm guessing the dependencies are due to **serialport**?

**Comment by emilyrose** [comment]
Yeah that's what I'm suspecting, but I didn't have the chance to capture all of the details from folks experiencing this output. It's fairly minor considering things still worked for everyone, but I don't like the idea of ugly warnings showing up when people install.

**Comment by emilyrose** [comment]
Thanks for checking on this, @kennethlimcp :smile_cat:
